### ISSUE TITLE
Please add a regression example

### ISSUE THREAD

**Issue by dan-lind** [rstudio/keras#235]
There are tons of great examples on https://keras.rstudio.com/articles/examples/index.html
But I can't seem to find a single regression example. 
It would make things so much easier if there was at least a single example with a "hello world", perhaps pointing out important differences for regression vs classification in terms of activations, loss functions, optimizers, metrics etc.

**Comment by jjallaire** [comment]
Agreed, will try to remedy this soon! In the meantime, this notebook provides a regression example based on the boston housing dataset: https://jjallaire.github.io/deep-learning-with-r-notebooks/notebooks/3.7-predicting-house-prices.nb.html

**Comment by dan-lind** [comment]
Great to hear! And thanks a lot for the link 😀

**Comment by raffscallion** [comment]
I'm also interested in a regression example. Unfortunately, the link posted above appears to be broken.

**Comment by raffscallion** [comment]
Wait, I found it.  Looks like the chapter number changed: [https://jjallaire.github.io/deep-learning-with-r-notebooks/notebooks/3.6-predicting-house-prices.nb.html](url)

**Comment by patwa67** [comment]
```
library(keras)
#Prepare mtcars data for Keras
data('mtcars')
mtcarsmat<-as.matrix(mtcars)
gearind <- to_categorical(as.numeric(mtcarsmat[,10])-3)
cylind <-to_categorical(as.numeric(mtcarsmat[,2])/2-2)
mtcarsx<-cbind(cylind,matrix(as.numeric(mtcarsmat[,3:9]),
              ncol=7),gearind,as.numeric(mtcarsmat[,11]))
y<-as.numeric(mtcarsmat[,1])
mtcarsxsc<-scale(mtcarsx)

#Multiple linear regression
modellr <- keras_model_sequential() 
modellr %>% 
  layer_dense(units = 1, input_shape = dim(mtcarsxsc)[2], activation='linear') %>%
  summary(modellr)
  early_stopping <- callback_early_stopping(monitor ='loss', min_delta=0.000001)
  
modellr %>% compile(
  loss = 'mean_squared_error',
  optimizer = optimizer_adam(lr = 0.001)
)

historylr <- modellr %>% fit(
  mtcarsxsc, y, 
  epochs = 1000, batch_size = 1,callbacks = early_stopping
)

plot(historylr)
lrweights <- get_weights(modellr)
plot(lrweights[[1]],xlab="Input variable",ylab="Weight",ylim=c(-1.5,1.5))

#LASSO without soft-thresholding
modell1 <- keras_model_sequential() 
modell1 %>% 
  layer_dense(units = 1, input_shape = dim(mtcarsxsc)[2], 
              kernel_regularizer=regularizer_l1(2.5),activation='linear') %>%
  summary(modell1)
early_stopping <- callback_early_stopping(monitor ='loss', min_delta=0.000001)

modell1 %>% compile(
  loss = 'mean_squared_error',
  optimizer = optimizer_adam(lr = 0.001)
)

historyl1 <- modell1 %>% fit(
  mtcarsxsc, y, 
  epochs = 1000, batch_size = 1,callbacks = early_stopping
)

plot(historyl1)
l1weights <- get_weights(modell1)
plot(l1weights[[1]],xlab="Input variable",ylab="Weight",ylim=c(-1.5,1.5))
```

**Comment by Steviey** [comment]
Is there an example with csv data?

**Comment by jjallaire** [comment]
Here's another regression tutorial: https://keras.rstudio.com/articles/tutorial_basic_regression.html
